FinanceTracker/FinanceTracker/Commands/AddOperationCommandBase.cs

```diff
@@ -11,9 +11,11 @@ public abstract class AddOperationCommandBase : ICommand
         public void Execute(Account account)
         {
             UserInputReader reader = new UserInputReader();
+            Guid id = Guid.NewGuid();
             decimal amount = reader.GetAmount();
             string description = reader.GetDescription();
-            Operation operation = new Operation(_type, amount, description);
+            string date = DateTime.Now.ToString("dd/MM/yyyy HH:mm");
```

Comment:
I would recommend to use DateTime.UtcNow, it will use no timezone. Its best practice at many BE application to avoid timezone issues, and you will apply it if needed separately

Reply:
Also you can move this format to const field